torch/csrc/jit/ir.cpp

```diff
@@ -639,7 +639,7 @@ std::shared_ptr<Graph> Graph::copy() {
 }
 
 bool Value::mustBeNone() const {
-  return node_->mustBeNone();
+  return type() == NoneType::get() || node_->mustBeNone();
```

Comment:
Why was this needed ?

Reply:
Because of the "bad breakage" above, we can't check for None parameters in `node_->mustBeNone`.
For parameters, you have (in general) more than one output.

Reply:
Gone now. :)